### ISSUE TITLE
[BUG] Longhorn says "invalid date"

### ISSUE THREAD

**Issue by Jellyfrog** [harvester/harvester#1364]
**Describe the bug**
![image](https://user-images.githubusercontent.com/759887/136271251-730e296c-1a72-4c09-b776-691f1ec69c3b.png)

**To Reproduce**
1. Create new VM with disk
2. Check longhorn storage

**Expected behavior**

**Support bundle**

**Environment:**
 - Harvester ISO version: Nightly from 2021-10-06
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**

I'm guessing the problem is the date format here: `"created": "2021-10-06 19:10:44 +0000 UTC",`

Json response from longhorn:
```
{
	"createTypes": {
		"volume": "https://10.48.11.70/v1/volumes"
	},
	"data": [{
		"accessMode": "rwx",
		"actions": {
			"activate": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=activate",
			"attach": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=attach",
			"cancelExpansion": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=cancelExpansion",
			"detach": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=detach",
			"engineUpgrade": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=engineUpgrade",
			"pvCreate": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=pvCreate",
			"pvcCreate": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=pvcCreate",
			"recurringJobAdd": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=recurringJobAdd",
			"recurringJobDelete": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=recurringJobDelete",
			"recurringJobList": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=recurringJobList",
			"replicaRemove": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=replicaRemove",
			"snapshotBackup": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=snapshotBackup",
			"snapshotCreate": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=snapshotCreate",
			"snapshotDelete": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=snapshotDelete",
			"snapshotGet": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=snapshotGet",
			"snapshotList": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=snapshotList",
			"snapshotPurge": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=snapshotPurge",
			"snapshotRevert": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=snapshotRevert",
			"updateDataLocality": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=updateDataLocality",
			"updateReplicaAutoBalance": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=updateReplicaAutoBalance",
			"updateReplicaCount": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca?action=updateReplicaCount"
		},
		"backingImage": "",
		"backupStatus": [],
		"cloneStatus": {
			"snapshot": "",
			"sourceVolume": "",
			"state": ""
		},
		"conditions": {
			"restore": {
				"lastProbeTime": "",
				"lastTransitionTime": "2021-10-06T19:10:46Z",
				"message": "",
				"reason": "",
				"status": "False",
				"type": "restore"
			},
			"scheduled": {
				"lastProbeTime": "",
				"lastTransitionTime": "2021-10-06T19:12:11Z",
				"message": "",
				"reason": "ReplicaSchedulingFailure",
				"status": "False",
				"type": "scheduled"
			},
			"toomanysnapshots": {
				"lastProbeTime": "",
				"lastTransitionTime": "2021-10-06T19:10:46Z",
				"message": "",
				"reason": "",
				"status": "False",
				"type": "toomanysnapshots"
			}
		},
		"controllers": [{
			"actualSize": "1400107008",
			"address": "10.52.0.200",
			"currentImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"endpoint": "/dev/longhorn/pvc-461521ea-b000-43b0-87c0-7450196554ca",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"hostId": "rancher06",
			"instanceManagerName": "instance-manager-e-67492476",
			"isExpanding": false,
			"lastExpansionError": "",
			"lastExpansionFailedAt": "",
			"lastRestoredBackup": "",
			"name": "pvc-461521ea-b000-43b0-87c0-7450196554ca-e-ab1406b5",
			"requestedBackupRestore": "",
			"running": true,
			"size": "21474836480"
		}],
		"created": "2021-10-06 19:10:44 +0000 UTC",
		"currentImage": "longhornio/longhorn-engine:v1.2.1-rc2",
		"dataLocality": "disabled",
		"dataSource": "",
		"disableFrontend": false,
		"diskSelector": null,
		"encrypted": false,
		"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
		"fromBackup": "",
		"frontend": "blockdev",
		"id": "pvc-461521ea-b000-43b0-87c0-7450196554ca",
		"kubernetesStatus": {
			"lastPVCRefAt": "",
			"lastPodRefAt": "",
			"namespace": "default",
			"pvName": "pvc-461521ea-b000-43b0-87c0-7450196554ca",
			"pvStatus": "Bound",
			"pvcName": "test-disk-1-9yxuz",
			"workloadsStatus": [{
				"podName": "virt-launcher-test-gnrzb",
				"podStatus": "Running",
				"workloadName": "test",
				"workloadType": "VirtualMachineInstance"
			}]
		},
		"lastAttachedBy": "",
		"lastBackup": "",
		"lastBackupAt": "",
		"links": {
			"self": "https://10.48.11.70/v1/volumes/pvc-461521ea-b000-43b0-87c0-7450196554ca"
		},
		"migratable": true,
		"name": "pvc-461521ea-b000-43b0-87c0-7450196554ca",
		"nodeSelector": null,
		"numberOfReplicas": 3,
		"purgeStatus": [{
			"actions": null,
			"error": "",
			"isPurging": false,
			"links": null,
			"progress": 0,
			"replica": "pvc-461521ea-b000-43b0-87c0-7450196554ca-r-81d715cc",
			"state": ""
		}],
		"ready": true,
		"rebuildStatus": [],
		"recurringJobSelector": null,
		"replicaAutoBalance": "ignored",
		"replicas": [{
			"address": "",
			"currentImage": "",
			"dataPath": "replicas",
			"diskID": "",
			"diskPath": "",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"failedAt": "",
			"hostId": "",
			"instanceManagerName": "",
			"mode": "",
			"name": "pvc-461521ea-b000-43b0-87c0-7450196554ca-r-42f00e5a",
			"running": false
		}, {
			"address": "",
			"currentImage": "",
			"dataPath": "replicas",
			"diskID": "",
			"diskPath": "",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"failedAt": "",
			"hostId": "",
			"instanceManagerName": "",
			"mode": "",
			"name": "pvc-461521ea-b000-43b0-87c0-7450196554ca-r-810e27d4",
			"running": false
		}, {
			"address": "10.52.0.201",
			"currentImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"dataPath": "/var/lib/harvester/extra-disks/695157b40b4a7c90f1c242384b0ee04e/replicas/pvc-461521ea-b000-43b0-87c0-7450196554ca-b86ea9f0",
			"diskID": "a364dae0-a711-4662-a931-e46429477d2e",
			"diskPath": "/var/lib/harvester/extra-disks/695157b40b4a7c90f1c242384b0ee04e",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"failedAt": "",
			"hostId": "rancher06",
			"instanceManagerName": "instance-manager-r-1214c321",
			"mode": "RW",
			"name": "pvc-461521ea-b000-43b0-87c0-7450196554ca-r-81d715cc",
			"running": true
		}],
		"restoreRequired": false,
		"restoreStatus": [{
			"actions": null,
			"backupURL": "",
			"error": "",
			"filename": "",
			"isRestoring": false,
			"lastRestored": "",
			"links": null,
			"progress": 0,
			"replica": "pvc-461521ea-b000-43b0-87c0-7450196554ca-r-81d715cc",
			"state": ""
		}],
		"revisionCounterDisabled": false,
		"robustness": "degraded",
		"shareEndpoint": "",
		"shareState": "",
		"size": "21474836480",
		"staleReplicaTimeout": 30,
		"standby": false,
		"state": "attached",
		"type": "volume"
	}, {
		"accessMode": "rwx",
		"actions": {
			"activate": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=activate",
			"attach": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=attach",
			"cancelExpansion": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=cancelExpansion",
			"detach": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=detach",
			"engineUpgrade": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=engineUpgrade",
			"expand": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=expand",
			"pvCreate": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=pvCreate",
			"pvcCreate": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=pvcCreate",
			"recurringJobAdd": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=recurringJobAdd",
			"recurringJobDelete": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=recurringJobDelete",
			"recurringJobList": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=recurringJobList",
			"replicaRemove": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=replicaRemove",
			"updateAccessMode": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=updateAccessMode",
			"updateDataLocality": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=updateDataLocality",
			"updateReplicaAutoBalance": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0?action=updateReplicaAutoBalance"
		},
		"backingImage": "default-image-csvmt",
		"backupStatus": [],
		"cloneStatus": {
			"snapshot": "",
			"sourceVolume": "",
			"state": ""
		},
		"conditions": {
			"restore": {
				"lastProbeTime": "",
				"lastTransitionTime": "2021-10-06T19:08:47Z",
				"message": "",
				"reason": "",
				"status": "False",
				"type": "restore"
			},
			"scheduled": {
				"lastProbeTime": "",
				"lastTransitionTime": "2021-10-06T19:37:26Z",
				"message": "Reset schedulable due to allow volume creation with degraded availability",
				"reason": "",
				"status": "True",
				"type": "scheduled"
			},
			"toomanysnapshots": {
				"lastProbeTime": "",
				"lastTransitionTime": "2021-10-06T19:08:47Z",
				"message": "",
				"reason": "",
				"status": "False",
				"type": "toomanysnapshots"
			}
		},
		"controllers": [{
			"actualSize": "0",
			"address": "",
			"currentImage": "",
			"endpoint": "",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"hostId": "",
			"instanceManagerName": "",
			"isExpanding": false,
			"lastExpansionError": "",
			"lastExpansionFailedAt": "",
			"lastRestoredBackup": "",
			"name": "pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0-e-16bc11bc",
			"requestedBackupRestore": "",
			"running": false,
			"size": "10737418240"
		}],
		"created": "2021-10-06 19:08:44 +0000 UTC",
		"currentImage": "longhornio/longhorn-engine:v1.2.1-rc2",
		"dataLocality": "disabled",
		"dataSource": "",
		"disableFrontend": false,
		"diskSelector": null,
		"encrypted": false,
		"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
		"fromBackup": "",
		"frontend": "blockdev",
		"id": "pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0",
		"kubernetesStatus": {
			"lastPVCRefAt": "",
			"lastPodRefAt": "2021-10-06T19:37:20Z",
			"namespace": "default",
			"pvName": "pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0",
			"pvStatus": "Bound",
			"pvcName": "test-disk-0-yyn9t",
			"workloadsStatus": [{
				"podName": "virt-launcher-test-xjpx4",
				"podStatus": "Failed",
				"workloadName": "test",
				"workloadType": "VirtualMachineInstance"
			}]
		},
		"lastAttachedBy": "",
		"lastBackup": "",
		"lastBackupAt": "",
		"links": {
			"self": "https://10.48.11.70/v1/volumes/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0"
		},
		"migratable": true,
		"name": "pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0",
		"nodeSelector": null,
		"numberOfReplicas": 3,
		"purgeStatus": null,
		"ready": true,
		"rebuildStatus": [],
		"recurringJobSelector": null,
		"replicaAutoBalance": "ignored",
		"replicas": [{
			"address": "",
			"currentImage": "",
			"dataPath": "replicas",
			"diskID": "",
			"diskPath": "",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"failedAt": "2021-10-06T19:37:25Z",
			"hostId": "",
			"instanceManagerName": "",
			"mode": "",
			"name": "pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0-r-02aa1fb5",
			"running": false
		}, {
			"address": "",
			"currentImage": "",
			"dataPath": "/var/lib/harvester/extra-disks/695157b40b4a7c90f1c242384b0ee04e/replicas/pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0-3b9eed86",
			"diskID": "a364dae0-a711-4662-a931-e46429477d2e",
			"diskPath": "/var/lib/harvester/extra-disks/695157b40b4a7c90f1c242384b0ee04e",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"failedAt": "",
			"hostId": "rancher06",
			"instanceManagerName": "",
			"mode": "",
			"name": "pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0-r-5da8b0a2",
			"running": false
		}, {
			"address": "",
			"currentImage": "",
			"dataPath": "replicas",
			"diskID": "",
			"diskPath": "",
			"engineImage": "longhornio/longhorn-engine:v1.2.1-rc2",
			"failedAt": "2021-10-06T19:37:25Z",
			"hostId": "",
			"instanceManagerName": "",
			"mode": "",
			"name": "pvc-b0ec0f32-4b2e-407d-bd75-2c74b026ccc0-r-f8e5f272",
			"running": false
		}],
		"restoreRequired": false,
		"restoreStatus": [],
		"revisionCounterDisabled": false,
		"robustness": "unknown",
		"shareEndpoint": "",
		"shareState": "",
		"size": "10737418240",
		"staleReplicaTimeout": 30,
		"standby": false,
		"state": "detached",
		"type": "volume"
	}],
	"links": {
		"self": "https://10.48.11.70/v1/volumes"
	},
	"resourceType": "volume",
	"type": "collection"
}
```

**Comment by yasker** [comment]
Ref: https://github.com/longhorn/longhorn/issues/3105

**Comment by smallteeths** [comment]
I found the reason, different browsers support different time formats and I need to convert the utc time format from golang. 

Google supports it, Firefox does not. cc @innobead

**Comment by smallteeths** [comment]
Ref: longhorn/longhorn#3230

**Comment by Jellyfrog** [comment]
This has been resolved in 1.0.0
